### DISCUSSION TITLE
Malware Spam

### DISCUSSION THREAD

**Discussion by whysloth** [community/community#119642] [Other Feature Feedback, Questions, & Ideas]
Today I discovered that an unknown group of people have registered dozens of accounts that have created a total of dozens of public repositories containing malware. The tags of these repositories contain references to hacking of Adobe and Microsoft software products. Examples of these repositories can be found in the hwid-changer topic:
https://github.com/topics/hwid-changer

I hope the repositories and users will be removed from the platform as soon as possible. I did not find a separate section on the forum for discussing security, so I created a discussion in the general section.

**Comment by Akash1134** [comment]
Hi @whysloth , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
